test/core/gen2/remote/test_config.py

```diff
@@ -40,12 +40,12 @@ def test_from_file(self):
         with open(file_path, "w") as fp:
             json.dump(json_instance, fp)
         try:
-            os.environ[
-                "_TEST_ENDPOINT_URL"
-            ] = "https://stage.xcube-gen.brockmann-consult.de/api/v2"
-            os.environ[
-                "_TEST_ACCESS_TOKEN"
-            ] = "02945ugjhklojg908ijr023jgbpij202jbv00897v0798v65472"
+            os.environ["_TEST_ENDPOINT_URL"] = (
+                "https://stage.xcube-gen.brockmann-consult.de/api/v2"
```

Comment:
If this is just a dummy URL, it's fine; but the URL is unreachable for years now. (I know, also not part of this PR)

Reply:
It is a dummy.